backend/api/chat_orchestrator.py

```
@@ -564,8 +596,10 @@ def _get_turns_remaining(self) -> int:
             return 0
         
         scene = self.scenes[self.state.current_scene_index]
-        # Use timeout_turns first, fallback to max_turns, with a reasonable default
-        timeout_turns = scene.get('timeout_turns') or scene.get('max_turns', 15)
+        # Use timeout_turns, require it to be set
+        timeout_turns = scene.get('timeout_turns')
+        if not timeout_turns:
+            raise ValueError("Scene must have timeout_turns configured")
```

Ruff (0.14.1)

Avoid specifying long messages outside the exception class

(TRY003)